scripts/server.py

```diff
@@ -0,0 +1,52 @@
+import traceback
+
+from flask import abort, Flask, request, jsonify
+
+from wikithingsdb import query, util
+
+app = Flask(__name__)
+
+
+@app.route("/types", methods=['GET', 'POST'])
+def get_types():
+    article = request.values.get('article', '').decode('utf-8')
+    methods = request.values.getlist('methods')
+
+    if not article:
+        abort(400)
+
+    response = {}
+    for function in methods:
+        try:
+            # calls query.function(article)
+            returned = getattr(query, function)(article)
+            response[function] = postprocess(returned, function)
+        except:
+            traceback.print_exc()
+
+    return jsonify(**response)
```

Comment:
jsonify takes keyword arguments. For instance, `jsonify(foo='bar', baz='qux')` produces `{'foo':'bar','baz':'qux'}`. `**response` unpacks response (a dictionary) into keyword arguments.